processor/src/test/resources/fixtures/org/mapstruct/ap/test/bugs/_1170/AdderSourceTargetMapperImpl.java

```diff
@@ -0,0 +1,150 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.test.bugs._1170;
+
+import java.math.BigDecimal;
+import java.util.List;
+import javax.annotation.Generated;
+import org.mapstruct.ap.test.bugs._1170._target.Target;
+import org.mapstruct.ap.test.bugs._1170.source.Source;
+import org.mapstruct.factory.Mappers;
+
+@Generated(
+    value = "org.mapstruct.ap.MappingProcessor",
+    date = "2017-05-21T21:54:47+0200",
+    comments = "version: , compiler: javac, environment: Java 1.8.0_131 (Oracle Corporation)"
+)
+public class AdderSourceTargetMapperImpl implements AdderSourceTargetMapper {
```

Comment:
The test fails because this file is wrong. I think you used an older master to test it. We no longer do a `null` per default. Also I am not sure if we really need a fixture here. We usually use fixtures when we want to test that the generated code looks like we are expecting. @gunnarmorling, @agudian, @sjaakd do you think that we need one here?